pom.xml

```diff
@@ -24,9 +24,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.nirima.docker-java</groupId>
+            <groupId>com.github.docker-java</groupId>
             <artifactId>docker-java</artifactId>
-            <version>0.10.5-SNAPSHOT</version>
+            <version>0.10.5</version>
```

Comment:
This wrong, SNAPSHOT version for not released plugin version is ok, because it needs fixes for functionality.